webhook.rb

```diff
@@ -33,6 +39,18 @@ class App < Sinatra::Base
     end
   end
 
+  def self.common_file?(file)
+    file =~ /\.github\/workflows\// && file !~ /\.github\/workflows\/.*-(rubygems|bundler)\.yml/
+  end
+
+  def self.bundler_file?(file)
+    file =~ /bundler\// || file == ".rubocop_bundler.yml" || file = /\.github\/workflows\/.*-bundler\.yml/
```

Comment:
We are safe here unless we plan to add `lib/bundler.rb` 😆

Reply:
To rubygems? I don't think that will happen any time soon, altough it's an idea for a clean path for merging the libraries :smile:.